qutip/core/options.py

```diff
@@ -25,8 +25,8 @@ class CoreOptions:
         general relative tolerance
         Used to choose QobjEvo.expect output type
 
-    auto_tidyup_atol : float {1e-12}
-        use auto tidyup absolute tolerance
+    auto_tidyup_atol : float {1e-14}
+        The absolute tolerance used in automatic tidyup.
```

Comment:
This is also the default value used in explicit calls to `Qobj.tidyup` so we should mention that in the docstring too.